### DISCUSSION TITLE
Hey Everyone!

### DISCUSSION THREAD

**Discussion by JakeC007** [OpenMined/courses#110] [Introduce Yourself]
Hi there!

I'm an undergraduate AI ethics researcher who is super jazzed about the intersection of AI, privacy, and policy.

I'm very excited about this course :)

_Jake

**Comment by yemikifouly** [comment]
Hey Jake! 🙂 Oh wow! As an AI ethics researcher, you will definitely enjoy this course very much! 😃 Welcome to the discussion board!
